ww4ff/run/demo.py

```diff
@@ -50,11 +50,11 @@ def _on_audio(self, in_data, frame_count, time_info, status):
         data_ok = (in_data, pyaudio.paContinue)
         self.last_data = in_data
         self._audio_buf.append(in_data)
-        if len(self._audio_buf) != 12:
+        if len(self._audio_buf) != 16:
             return data_ok
         audio_data = b''.join(self._audio_buf)
         self._audio_buf = self._audio_buf[2:]
-        arr = np.frombuffer(audio_data, dtype=np.int16).astype(np.float) / 32768
+        arr = 1.41 * (np.frombuffer(audio_data, dtype=np.int16).astype(np.float) / 32767)
```

Comment:
can you explain what this change is for?

Reply:
Ugh, the 1.41 is a bug. I'll fix it.